ledger-service/http-json-cli/src/main/scala/com/daml/http/SchemaHandling.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http
+import scalaz.std.option._
+import scalaz.syntax.traverse._
+
+private[http] sealed trait SchemaHandling
+private[http] object SchemaHandling {
+  private[http] case object ForceCreateAndTerminate extends SchemaHandling
+  private[http] case object CheckAndTerminateIfWrong extends SchemaHandling
+  private[http] case object CreateOrUpdateAndContinue extends SchemaHandling
+  private[http] case object ForceCreateAndContinue extends SchemaHandling
```

Comment:
```suggestion
  private[http] case object CreateSchema extends SchemaHandling
  private[http] case object Start extends SchemaHandling
  private[http] case object CreateSchemaIfNeededAndStart extends SchemaHandling
  private[http] case object CreateSchemaAndStart extends SchemaHandling
```
seems a bit clearer to me

Reply:
I'm not super happy with `Start`, it sounds a bit weird to me in the config. I think `JustStart` somehow sounds sensible, although it is still not the best naming. :thinking: :confused:

Reply:
I chose the names to make it clear that CreateSchemaAndStart is exactly doing first CreateSchema and then Start but I don’t feel all that strongly about the specific names.

Reply:
Well let's go with it. If someone comes up with better names we can change it before the next release.